### PR TITLE
[GH-121] Add info to Zoom OAuth UI

### LINKED ISSUE #121: Add info to Zoom oAuth UI
We should add info to the Zoom configuration screen for Oauth

1. Add `Beta` label beside oAuth config enable option
2. There is currently a label that reads `Please enable only either one of OAuth based or Password base authentication.`  
This should be updated to also explain about the mapping. Please change it to `Please enable only either OAuth based or Password base authentication. If you are currently using a JWT Zoom application and switch to oAuth, users will need to connect again using oAuth the first time they create a meeting.`

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Add info to Zoom OAuth UI on System Console

#### Ticket Link
Fix #121

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -35,9 +35,9 @@
             },
             {
                 "key": "EnableOAuth",
-                "display_name": "Enable OAuth",
+                "display_name": "Enable OAuth (Beta)",
                 "type": "bool",
-                "help_text": "When true, OAuth will be used as authentication means with Zoom. \n Please enable only either one of OAuth based or Password base authentication.",
```
Comment (jfrerich) [inline plugin.json:40]: If requiring only one of Oauth or Password based authentication, I feel we should account for that in code and not rely on the admin to make sure this happens.

**Comment by terafirmanz** [comment]
This could be done with: Use OAuth for Authentication? if true use OAuth if false then use password. This means there is no way to enable both.

File: plugin.json (hunk 1 above)
Comment (larkox) [reply plugin.json:40]: It is also taken care in the code (the plugin should not enable if both are checked). Nevertheless, it is important to make sure the admin understand why it is failing.

**Comment by larkox** [comment]
@DHaussermann Just system console texts changed.

**Comment by DHaussermann** [comment]
@larkox since we're changing this. There is a minor grammar error in the existing text where `the` is missing.

Can you please change `When true, OAuth will be used as authentication means with Zoom.` to `When true, OAuth will be used as the authentication means with Zoom.`

**Pull request merged**